im-gespraech-buber-gesellschaft.csl

```diff
@@ -0,0 +1,425 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0" demote-non-dropping-particle="sort-only">
+  <info>
+    <title>Im Gespräch - Hefte der Martin Buber-Gesellschaft (German)</title>
```

Comment:
@adam3smith, how do you feel about having the subtitle in the title here? I usually try to shy away from that.